com.unity.render-pipelines.high-definition/Editor/Lighting/LightUnit/LightUnitSlider.cs

```diff
@@ -261,24 +261,27 @@ struct Piece
         {
             public Vector2 domain;
             public Vector2 range;
-            public Func<float, float> transform;
-            public Func<float, float> inverseTransform;
+
+            public float directM;
+            public float directB;
+            public float inverseM;
+            public float inverseB;
         }
 
         // Piecewise function indexed by value ranges.
-        private readonly Dictionary<int, Piece> m_PiecewiseFunctionMap = new Dictionary<int, Piece>();
+        private readonly Dictionary<Vector2, Piece> m_PiecewiseFunctionMap = new Dictionary<Vector2, Piece>();
 
-        Func<float, float> GetTransformation(float x0, float x1, float y0, float y1)
+        static void NewTransformation(float x0, float x1, float y0, float y1, out float m, out float b)
```

Comment:
Now, the naming is still incorrect for other reason. `NewTransformation` was a good choice when it instantiated a lambda, now it would be more `ComputeTransformationParameters`.

I think that I had a confusing request in the previous review, the `NewTransformation` naming was good with the lambda pattern, but now we use directly an algorithm with parameters, the naming should be different.

Sorry about that.